lib/index.js

```diff
@@ -8,8 +8,10 @@ let { abort, repr } = require("./util");
 let SerializedRunner = require("./util/runner");
 let browserslist = require("browserslist");
 
-module.exports = (referenceDir, config,
-		{ watch, fingerprint, sourcemaps, compact, serve, liveserve }) => {
+module.exports = async function faucetDispatch(referenceDir, config,
```

Comment:
I'm confused. Don't we need to add a `await` somewhere, where we call `faucetDispatch`?

Reply:
We don't, FND explained that nobody cares about the return value of this function